src/locales/en.json

```diff
@@ -125,34 +127,41 @@
     "about_para": "The PathCheck app is made available by the PathCheck Foundation, a non-profit organization that is committed to providing free software to help stop the spread of COVID-19 and protect the privacy of individuals. \n\nFor more information, visit:",
     "assessment_icon": "Clipboard icon with checkmark inside",
     "bell_icon": "Bell icon",
-    "bluetooth_icon": "Searching bluetooth device icon",
     "calendar_icon": "Calendar icon",
     "check": "Checkmark",
     "checkbox": "Checkbox",
     "close": "Close",
     "description_optional": "Description (Optional)",
     "email_optional": "Email (Optional)",
     "exposure_icon": "Circle icon with eight dots surrounding it",
-    "heart_icon": "Heart icon with cross inside",
     "home_icon": "Home icon",
     "language_icon": "Outlined icon of a globe",
-    "launch_disable_exposure_notif": "Don't Enable",
-    "launch_enable_exposure_notif": "Enable",
     "launch_enable_notif": "Enable Notifications",
-    "launch_exposure_notif_header": "Enable Exposure Notifications",
-    "launch_exposure_notif_subheader": "To be notified about updates to your exposure history, your phone needs permission to exchange anonymous Bluetooth random ID with other devices.\n\nExposure Notifications only work with other devices with the setting enabled.\n\nYou can turn off this notifications at any time.",
     "launch_get_started": "Get Started",
     "launch_next": "Next",
     "launch_screen1_header": "Welcome to ",
-    "launch_screen2_header_bluetooth": "Your phone remembers other devices it meets but won't identify you to other people.",
-    "launch_screen2_subheader_bluetooth": "Nearby phones use Bluetooth to share and save random number strings to recall devices nearby.\n\nIf a person reports a positive diagnosis, their phone's recent random IDs are flagged to anonymously notify others. No locations or personal info is shared.",
     "launch_screen3_header_bluetooth": "Receive notifications when someone you encountered later chooses to report a positive diagnosis.",
     "launch_screen3_subheader_bluetooth": "Learn details about the encounter, such as the date, duration, and Bluetooth signal strength.",
     "launch_screen4_header_bluetooth": "If you test positive for the virus that causes COVID-19, you can choose to share your diagnosis anonymously.",
     "launch_screen4_subheader_bluetooth": "This helps others in your community contain the spread of the virus.",
     "launch_set_up_phone_bluetooth": "Next",
     "legal_page_address": "PathCheck Foundation\n58 Day Street\nBox 441621\nSomerville, MA 02144",
     "more_icon": "Icon of three horizontal dots",
+    "onboarding_screen1_header": "Together, we can contain the spread of COVID-19",
```

Comment:
These should probably be moved to the `onboarding` section.

Reply:
there is an automated script to handle some of this moving around, let me know if you want to pair on it.